slides/testing/tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
I am going to use the extend feature of the tsconfig to create a base config file and then extend it into a new file for cirecle ci to set the watch property to false.